engine/src/em-event.js

```diff
@@ -508,11 +532,25 @@ mergeInto(LibraryManager.library, {
 						 [stack, modifiers, char_code, key_code]);
 		},
 
-		_handleKeyboardEvent: function(e) {
+		_handleKeyboardEvent: function(e, pCanvas) {
 			LiveCodeAsync.delay(function() {
 
-				var stack = LiveCodeEvents._getStackForCanvas(e.target);
-				var mods = LiveCodeEvents._encodeModifiers(e);
+				var stack = LiveCodeEvents._getStackForCanvas(pCanvas);
+				/* TODO - reenable alt key detection
```

Comment:
Does your other pull request for alt key detection fix this 'Todo' so that you can remove the note?

Reply:
@seaniepie no, it's a trade-off to allow extended character input using alt key combinations to work. I'm still looking for a way to re-enable it.